build.gradle.kts

```diff
@@ -76,6 +76,8 @@ subprojects {
                 showExceptions = true
                 showCauses = true
                 showStackTraces = true
+                // TODO: add the other envvars that can trigger debug output
+                showStandardStreams = System.getenv("MODS_ELN_DEBUG") != ""
```

Comment:
We have more than just this. We could detect all of them by hooking into the debug code? Perhaps for another PR. Let's keep this PR just to CG.

Reply:
I needed it for this PR, so I stuck it in there for only that reason. I can totally back it out though :P

Reply:
I guess it's fine, but flesh it out I guess?